server/src/util.rs

```diff
@@ -0,0 +1,17 @@
+//! Utilities. Functions that do not belong anywhere else.
+use crate::protocol::client_protocol::GeoPosition;
+use crate::protocol::server_protocol::{ServerOutput, Vehicle, VehiclePositionsOutput};
+use geoutils::Location;
+
+///Returns a bool representing whether a vehicle is close enough to be sent out.
+///To be used with a filter function.
+pub fn filter_position(client_geo: &GeoPosition, vhc: &Vehicle) -> bool {
```

Comment:
Would be good to rename this to `filter_vehicle_positions` or something similar, to make it clear what type of positions we are filtering.